.github/workflows/ci.yml

```diff
@@ -148,3 +145,21 @@ jobs:
         env:
           AWX_TEST_IMAGE: awx
           AWX_TEST_VERSION: ci
+
+  collection-sanity:
+    name: awx_collection sanity
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Upgrade ansible-core
```

Comment:
Isn't this installing it in the first place?

Reply:
The Ubuntu GHA runner has Ansible installed by default.